server/services/mqtt/lib/setValue.js

```diff
@@ -0,0 +1,18 @@
+/**
+ * @description Set value.
+ * @param {Object} device - The device to control.
+ * @param {Object} deviceFeature - The device feature to control.
+ * @param {number} value - The value to set.
+ * @example
+ * mqtt.setValue();
+ */
+function setValue(device, deviceFeature, value) {
+  const originalExternalId = deviceFeature.external_id;
+  // Removes 'mqtt:'
+  const externalId = originalExternalId.substring(5, originalExternalId.length);
+  this.publish(`gladys/${externalId}/update`, value);
```

Comment:
This MQTT topic doesn't make a lot of sense! 

MQTT topics should be chosen carefully, like a REST API. 

If I understand the discussion on the forum, you're developing this to allow `Terdious` to control his wide range of Arduino through MQTT.

I think we need to have a talk on this before. This is a "Gladys Pod" discussion!